app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -120,12 +126,8 @@ class PocketVideoFetchScheduler(
         @VisibleForTesting(otherwise = PRIVATE) val BACKOFF_DELAY_MIN_MILLIS = TimeUnit.SECONDS.toMillis(30)
         @VisibleForTesting(otherwise = PRIVATE) val BACKOFF_DELAY_MAX_MILLIS = TimeUnit.SECONDS.toMillis(60)
 
-        private var DELAY_TIME_FOR_QA_MILLIS = TimeUnit.SECONDS.toMillis(5)
+        private var DELAY_TIME_FOR_QA_MILLIS: Long? = null
```

Comment:
These variables should be moved into the class body, otherwise this is still static state.